docs/tools/fbsql/fbsql-loaders.md

```diff
@@ -0,0 +1,53 @@
+---
+title: fbsql-loaders
+layout: default
+parent: fbsql
+grand_parent: Tools
+nav_order: 2
+---
+
+# FBSQL Loaders
```

Comment:
Same here -- what are the loaders for? Example title could be:

How do I load data from external datasources using fbsql?